### PR TITLE
Update OMO to latest ODK 1.5.3

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: docs/odk-workflows/ManageDocumentation.md [hunk 1]
```diff
@@ -38,9 +38,10 @@ The documentation is _not_ automatically updated from the Markdown, and needs to
    If everything was successful, you will see a message similar to this one:
 
    ```
-   INFO    -  Your documentation should shortly be available at: https://information-artifact-ontology.github.io/ontology-metadata/ 
+   INFO    -  Your documentation should shortly be available at: https://information-artifact-ontology.github.io/noname/
```
Comment (anitacaron) [inline docs/odk-workflows/ManageDocumentation.md:41]: What did happen with the ontology name?

**Review by anitacaron — CHANGES_REQUESTED** [review]

File: docs/odk-workflows/ManageDocumentation.md (hunk 1 above)
Comment (anitacaron) [reply docs/odk-workflows/ManageDocumentation.md:41]: Are you going to discard the changes on this and ReleaseWorkflow files?

Comment (matentzn) [reply docs/odk-workflows/ManageDocumentation.md:41]: ooooookeeeeeeee...

Comment (anitacaron) [reply docs/odk-workflows/ManageDocumentation.md]: Thank you! 😊

**Pull request merged**